core/src/test/java/org/hibernate/ogm/backendtck/storedprocedures/NamedParametersStoredProcedureCallTest.java

```diff
@@ -174,6 +174,10 @@ public void testExceptionWhenProcedureDoesNotExist() throws Exception {
 		storedProcedureQuery.getSingleResult();
 	}
 
+	@SkipByGridDialect(
+			value = { INFINISPAN_REMOTE },
+			comment = "This dialects does not throw exception when using not registered parameters."
+	)
```

Comment:
We cannot determine the invalid parameter before sending it to the execution because it's optional for Infinispan. However, we can analyze the execution result and extract the parameter name. Should we implement it?

Reply:
yes, I think we should do that. Unless, it's too complicated